src/app/public/components/impressum/impressum.component.html

```diff
@@ -5,66 +5,57 @@
 		<h1 class="u-text-h1-bold u-margin-bottom2x u-text-dark2 u-margin-top4x">Impressum</h1>
 
 		<div>
-			<p style="font-size: 1.1rem; color: black; line-height: 1.55; margin: 1em 0 1.3em">
-				<span style="font-weight: bold" class="u-text u-margin-bottom2x u-text-dark2 u-margin-top4x">
-					openSenseLab gGmbH
-				</span>
-				<br />
-				Von-Steuben-Str. 21
-				<br />
-				48143 Münster
-				<br /><br />
+			<p style="font-size: 1.1rem; color: black; line-height: 1.55; margin: 2em 0 1.3em">
+				Open Educational Badges ist ein Gemeinschaftsprojekt von
+				<a href="https://matrix-gruppe.de/matrix-gemeinnuetzig/"> <u>matrix gGmbh</u> </a>,
```

Comment:
Can you make it more clear visually that this is a link? I think we use these classes for links: ```class='tw-underline tw-text-[#1400FF]'``` Also, i guess it should probably open a new tab

Reply:
I think it would be good to open the links here in a new tab, but i will approve